src/widget.tsx

```diff
@@ -248,73 +278,26 @@ export class LuxWidgetView extends DOMWidgetView {
       }
 
       render() {
-        let exportBtn;
```

Comment:
This is good that we're cleaning up the big widget file and moving them into components.